src/test/java/com/ghilly/controller/CountryControllerTest.java

```diff
@@ -12,70 +13,75 @@
 
 class CountryControllerTest {
 
-    private static final CountryServiceRest service = mock(CountryServiceRest.class);
-    private static final CountryController controller = new CountryController(service);
-    private static final int id = 100;
-    private static final String name = "USSR";
+    private CountryServiceRest service;
+    private CountryController controller;
+    private static final int ID = 100;
+    private static final String NAME = "USSR";
 
+    @BeforeEach
+    void init() {
+        service = mock(CountryServiceRest.class);
+        controller = new CountryController(service);
+    }
 
     @Test
     void createCountry() {
-        controller.create(name);
+        controller.create(NAME);
 
         assertAll(
-                () -> verify(service).add(name),
+                () -> verify(service).add(NAME),
                 () -> verifyNoMoreInteractions(service)
         );
     }
 
     @Test
     void getCountries() {
         List<String> expected = new ArrayList<>(List.of("Japan", "Russia", "Germany"));
```

Comment:
Why you use ArrayList, list.of return List

Reply:
+